include/seqan3/alignment/pairwise/policy/banded_score_dp_matrix_policy.hpp

```diff
@@ -91,6 +91,10 @@ class banded_score_dp_matrix_policy :
         band_row_index = std::abs(std::min(static_cast<int_fast32_t>(band.lower_bound),
                                            static_cast<int_fast32_t>(0)));
 
+        // If the band is wider than the sequence length, limit the band width.
+        band_column_index = std::min(band_column_index, static_cast<uint_fast32_t>(dimension_second_range - 1));
```

Comment:
This should actually not impose a problem. Why this change?

Reply:
It does impose a problem, because if the user specifies a band width larger than sequence length, the `band_column_index` is out of bounds. 
```console
Assertion failed: (current_column_index > band_column_index), function second_range_begin_offset, file /Users/jwinkler/repo/seqan3/include/seqan3/alignment/pairwise/policy/banded_score_dp_matrix_policy.hpp, line 174.
```
Furthermore, it will allocate a too large DP matrix a few lines below.

Reply:
Strange, I need to investigate this, because basically it should not matter.